### DISCUSSION TITLE
Recording Bitstreams

### DISCUSSION THREAD

**Discussion by mikkut54** [DSheirer/sdrtrunk#1693] [Q&A]
Regarding the options to record demodulated bitstreams and traffic channel demodulated bitstreams, just curious what can be done with those recordings and what app/program would you use to work with them.  Thanks!

**Comment by DSheirer** [comment]
There's a 'Message Recording Viewer' tool you can access in the SDRTrunk application under the 'View' menu.  That tool let's you view DMR and APCO25 Phase 1 bitstream recordings to see the individual messages.
